modules/local/rename_predastool.nf

```diff
@@ -0,0 +1,26 @@
+process RENAME_PREDASTOOL {
+    tag "${meta.assembler}-${meta.id}"
+    label 'process_low'
+
+    // Using container from multiqc since it'll be included anyway
+    conda (params.enable_conda ? "bioconda::multiqc=1.12" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/multiqc:1.12--pyhdfd78af_0' :
+        'quay.io/biocontainers/multiqc:1.12--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bins)
+
+    output:
+    tuple val(meta), path("${meta.assembler}-${meta.binner}Refined-${meta.id}*")           , optional:true, emit: renamed_bins
+
+    script:
+    """
+    for i in ${bins}; do
+        filename=\$i
+        suffix=\${filename#*.}
+
+        mv \$i ${meta.assembler}-${meta.binner}Refined-${meta.id}.\$suffix
```

Comment:
if other binners would name their bins differently, e.g. `*${meta.id}_\$suffix`, this would cause a problem, right? not sure, would it make sense to generalise and extract the bin name from the filename?

Reply:
Theoretically yes, but extracting the bin name is just as problematic as each file name could be completely different, so coming up with a global parser would be too complicated...

I would rather keep this like so for now and we address that problem later.